modules/upgrade.adoc

```diff
@@ -20,19 +20,23 @@ All Kubernetes objects and PVs in each {product-title} cluster are backed up as
 ====
 
 [id="upgrade-automatic_{context}"]
-== Automatic upgrades
+== Recurring upgrades
 
 Upgrades can be scheduled to occur automatically on a day and time specified by the cluster owner or administrator. Upgrades occur on a weekly basis, unless an upgrade is unavailable for that week.
 
+If you select recurring updates for your cluster, you must provide an administrator’s acknowledgment. {cluster-manager} does not start scheduled y-stream updates for minor versions without receiving an administrator’s acknowledgment. For information about administrator acknowledgment, see link:https://docs.openshift.com/dedicated/upgrading/osd-upgrading-cluster-prepare.html#upgrade-49-acknowledgement_osd-updating-cluster-prepare[Administrator acknowledgment when upgrading to OpenShift 4.9].
+
 [NOTE]
 ====
-Automatic upgrade policies are optional and if they are not set, the upgrade policies default to manual.
+Recurring upgrade policies are optional and if they are not set, the upgrade policies default to individual.
 ====
 
 [id="upgrade-manual_upgrades_{context}"]
-== Manual upgrades
+== Individual upgrades
+
+If you opt for individual upgrades, you are responsible for updating your cluster. If you select an update version that requires approval, you must provide an administrator’s acknowledgment. For information about administrator acknowledgment, see xref:./../upgrading/osd-upgrading-cluster-prepare.adoc#upgrade-49-acknowledgement_osd-updating-cluster-prepare[Administrator acknowledgment when upgrading to OpenShift 4.9].
 
-If you opt for manual upgrades, you are responsible for updating your cluster. If your cluster version falls too far behind, it will transition to a limited support status. For more information on OpenShift life cycle policies, see xref:../osd_policy/osd-life-cycle.adoc#osd-life-cycle[{product-title} update life cycle].
+If you opt for manual upgrades, you are responsible for updating your cluster. If your cluster version falls too far behind, it will transition to a limited support status. For more information on OpenShift life cycle policies, see xref:../osd_architecture/osd_policy/osd-life-cycle.adoc#osd-life-cycle[{product-title} update life cycle].
```

Comment:
Change "fall too far behind" to "becomes outdated"?

Reply:
Made the change.